spec/controllers/omniauth_callbacks_controller_spec.rb

```diff
@@ -85,5 +85,20 @@
       get :barcode
       expect(response).to redirect_to(account_path)
     end
+    context 'when a 429 response is received from the Alma API endpoint' do
+      before do
+        allow(User).to receive(:from_barcode) { valid_barcode_user }
+        allow(Bibdata).to receive(:get_patron).and_raise(Bibdata::PerSecondThresholdError)
+      end
+
+      it 'expired netid barcode redirects to account page' do
```

Comment:
This spec title doesn't look right. Maybe it should just be `it "redirects to account page"`